### PR TITLE
fix: missing default_argument component overriding and predicate-options display

### PR DESCRIPTION
**Pull request by davidlandais**
### Prerequisites
- [x] I have read the [Contributing guidelines](https://github.com/fgribreau/ui-predicate/blob/master/.github/CONTRIBUTING.md)
- [x] I have read the [Code of conduct](https://github.com/fgribreau/ui-predicate/blob/master/.github/CODE_OF_CONDUCT.md) and I agree with it

### Description
Minor fix to handle UITypes.DEFAULT_ARGUMENT component if override wanted.

### CODE REVIEW

File: packages/ui-predicate-core/src/PredicateCore.js [hunk 1]
```diff
@@ -568,7 +568,7 @@ module.exports = function({ dataclasses, invariants, errors, rules, UITypes }) {
             )
             .map(argumentType => argumentType.component)
             .valueOrElse(() =>
-              _options.getDefaultArgumentComponent(_columns, _options)
+              _options.getDefaultArgumentComponent(_columns, _options, ui)
```
Comment (FGRibreau) [inline packages/ui-predicate-core/src/PredicateCore.js]: should be _ui (see _options declaration)

File: packages/ui-predicate-vue/src/UIPredicateCoreVue.js [hunk 2]
```diff
@@ -15,21 +15,31 @@ const defaults = {
   options: {
     /**
      * UIPredicate Vue Adapter own default argument component
+     * @param {Object} [columns=PredicateCore.defaults.columns] columns
+     * @param {Object} [options=PredicateCore.defaults.options] options
+     * @param {Object} [ui=PredicateCore.defaults.ui] ui
```
Comment (FGRibreau) [inline packages/ui-predicate-vue/src/UIPredicateCoreVue.js]: :+1:

File: packages/ui-predicate-vue/src/UIPredicateCoreVue.js [hunk 3]
```diff
@@ -15,21 +15,31 @@ const defaults = {
...
 };
 
 /**
  * UIPredicateCore adapter for VueJS
- * @param       {?dataclasses.CompoundPredicate} [data=PredicateCore.defaults.options.getDefaultData] data
- * @param       {Object} [columns=PredicateCore.defaults.columns] columns
- * @param       {Object} [options=PredicateCore.defaults.options] options
+ * @param {?dataclasses.CompoundPredicate} [data=PredicateCore.defaults.options.getDefaultData] data
+ * @param {Object} [columns=PredicateCore.defaults.columns] columns
+ * @param {Object} [ui=PredicateCore.defaults.ui] ui
```
Comment (FGRibreau) [inline packages/ui-predicate-vue/src/UIPredicateCoreVue.js]: :+1:

File: packages/ui-predicate-vue/src/ui-predicate-options.vue [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-      :disabled="predicate.$canBeRemoved === false"
-    />
-    <component
-      :is="getUIComponent(UITypes.PREDICATE_ADD)"
-      @click.native="add(predicate)"
-      :predicate="predicate"
-      :is-in-add-compound-mode="isInAddCompoundMode"
-    />
+  <div class="ui-predicate__col">
+    <div class="ui-predicate__options">
```
Comment (FGRibreau) [inline packages/ui-predicate-vue/src/ui-predicate-options.vue]: no need, see https://github.com/FGRibreau/ui-predicate/commit/4b0aff896fbf7ebc4363055bd62d1e9f0d27c166

Comment (davidlandais) [reply packages/ui-predicate-vue/src/ui-predicate-options.vue]: On build, in ui-predicate-options.vue, if there is 2 sibling dynamic component, the last one is not rendered. So i need to wrap each other in a div. And so, i prefer to use a plural class name on parent and singular on children (wrapper).

**Comment by FGRibreau** [comment]
@davidlandais rebase, netlify build should pass now :)

**Pull request merged**